phoenix-core-client/src/main/java/org/apache/phoenix/parse/OrderByNode.java

```diff
@@ -84,4 +85,16 @@ public void toSQL(ColumnResolver resolver, StringBuilder buf) {
         if (!orderAscending) buf.append(" DESC");
         if (nullsLast) buf.append(" NULLS LAST ");
     }
+
+    public boolean isLiteral() {
```

Comment:
Maybe we could rename this method to `isIntegerLiteral`

Reply:
Ok, already fixed.